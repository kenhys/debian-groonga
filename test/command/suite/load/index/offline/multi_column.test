table_create Users TABLE_NO_KEY
column_create Users name COLUMN_SCALAR ShortText
column_create Users address COLUMN_SCALAR ShortText

load --table Users
[
["name", "address"],
["Alice", "France"],
["Bob", "England"]
]

table_create Words TABLE_PAT_KEY|KEY_NORMALIZE --key_type ShortText \
  --default_tokenizer TokenBigramSplitSymbolAlpha
column_create Words users_all COLUMN_INDEX|WITH_POSITION|WITH_SECTION \
  Users name,address

select Words --output_columns _key
