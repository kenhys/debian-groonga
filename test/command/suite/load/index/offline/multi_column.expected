table_create Users TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Users name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Users address COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Users
[
["name", "address"],
["Alice", "France"],
["Bob", "England"]
]
[[0,0.0,0.0],2]
table_create Words TABLE_PAT_KEY|KEY_NORMALIZE --key_type ShortText   --default_tokenizer TokenBigramSplitSymbolAlpha
[[0,0.0,0.0],true]
column_create Words users_all COLUMN_INDEX|WITH_POSITION|WITH_SECTION   Users name,address
[[0,0.0,0.0],true]
select Words --output_columns _key
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        18
      ],
      [
        [
          "_key",
          "ShortText"
        ]
      ],
      [
        "al"
      ],
      [
        "an"
      ],
      [
        "b"
      ],
      [
        "bo"
      ],
      [
        "ce"
      ],
      [
        "d"
      ],
      [
        "e"
      ],
      [
        "en"
      ],
      [
        "fr"
      ],
      [
        "gl"
      ]
    ]
  ]
]
